common-ktx/src/main/java/co/nimblehq/common/extensions/StringExt.kt

```diff
@@ -2,14 +2,22 @@ package co.nimblehq.common.extensions
 
 import androidx.core.util.PatternsCompat
 import java.util.*
+import kotlin.contracts.ExperimentalContracts
+import kotlin.contracts.contract
 
 /**
  * Check if the string not null or empty.
  * This extension wraps for more readable.
  *
  * @return true if this nullable char sequence is NOT either null or empty
  */
-fun String?.isNotNullOrEmpty(): Boolean = !this.isNullOrEmpty()
+@OptIn(ExperimentalContracts::class)
+fun String?.isNotNullOrEmpty(): Boolean {
+    contract {
+        returns(false) implies (this@isNotNullOrEmpty != null)
```

Comment:
I think this should be returns true since it's not null or empty
```suggestion
        returns(true) implies (this@isNotNullOrEmpty != null)
```